test/gtest/ucp/ucp_test.cc

```diff
@@ -114,6 +114,9 @@ void ucp_test::disconnect(const entity& entity) {
         entity.flush_worker(i);
         void *dreq = entity.disconnect_nb(i);
         if (!UCS_PTR_IS_PTR(dreq)) {
+            if (UCS_ERR_ENDPOINT_TIMEOUT == UCS_PTR_STATUS(dreq)) {
```

Comment:
why is this needed? does it happen only for err handling tests?
need to make sure we don't ignore errors on all other tests